app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -117,6 +117,17 @@ def hash_for_pdf
                                                  ]))
       end
 
+      # line 12
+      if @xml_document.at("Exemptions DependAttendCollege")
+        count = @xml_document.at("Exemptions DependAttendCollege").text.to_i
+        digits = count.digits
+        answers[:undefined_14] = digits[0]
+        answers[:'x  1000_4'] = count * 1_000
```

Comment:
i ran into a similar decision [on my upcoming PR](https://github.com/codeforamerica/vita-min/pull/5014/files#diff-2062992eaee6859ddbb1cc1b5096eeae8efbc7e474aa5ee72ef286e35618a239R129) (i haven't yet merged this work into mine to resolve conflicts), where I couldn't get a value for the PDF from XML because it wasn't in the XML. I ended up getting it from the calculator-- otherwise there are two sources of truth for the calculation

Reply:
I like the idea of getting this value from the calculator

Reply:
Let's talk about this one in our eng sync!

Reply:
Convo result: keep in this PR for now, Anne or Matt will do a fast follow PR to refactor all instances of this in the PDF to pull from calc instead